templates/prometheus.yaml

```diff
@@ -90,6 +94,25 @@ spec:
           thanos:
             baseImage: thanosio/thanos
             version: v0.4.0
+          externalLabels:
+            cluster: $(CLUSTER_ID)
+          containers:
+            # note: in order to run this container, mesosphereResources.enableAdditionalPrometheusRBACRules
+            # needs to be set to true
+            - name: get-cluster-id
+              image: mesosphere/kubeaddons-addon-initializer:v0.0.13
+              command: ["/bin/bash", "-c", "addon-initializer prometheus && sleep infinity"]
+              env:
+              - name: "PROMETHEUS_NAMESPACE"
+                value: "kubeaddons"
```

Comment:
What if we change the namespace? This should be more dynamic if we change the namespace name at any time

Reply:
Why cannot u get it as u do for the cluster_id?

Reply:
I think it's reasonable to have it static here - if we do change the namespace, this file would need to be updated in multiple locations since there are a bunch of references to the namespace (e.g. the metadata for this kubeaddon specifies `kubeaddons` ns statically)